elements/pl-string-input/pl-string-input.mustache

```diff
@@ -48,7 +48,7 @@
 {{/parse_error}}
 {{^parse_error}}
 {{#label}}<span>{{label}}</span>{{/label}}
-<samp>{{a_sub}}</samp>
+<samp style="white-space: pre; background-color: #e8e8e8;">{{a_sub}}</samp>
```

Comment:
It's best to avoid inline CSS styling, if possible.

1) Is the `white-space` style necessary?
2) Could you replace the `background-color` style with `class="bg-something"` as in https://getbootstrap.com/docs/4.0/utilities/colors/?

Reply:
The `whits-space: pre` is needed, otherwise by default, it uses `white-space: normal`, where "Sequences of whitespace will collapse into a single whitespace".

I don't particularly like the color options from the link you indicated. They are mostly to dark or distracting. `bg-light` is the only one I would choose, but I think that is a little too light, so I still prefer the `background-color` suggested by Matt.

Reply:
okay, I will add this soon and push again